feat(deps): version bump for 5.2.1 release (#3913)
patternfly/patternfly-org: packages/documentation-site/patternfly-docs/content/get-started/release-highlights.md

### Promoted package versions

- patternfly/patternfly ([changelog](https://github.com/patternfly/patternfly/releases/tag/patch-5.2.1))
  - [@patternfly/patternfly@5.2.1](https://www.npmjs.com/package/@patternfly/patternfly/v/5.2.1)

- patternfly/patternfly-react ([changelog](https://github.com/patternfly/patternfly-react/releases/tag/5.2.1))
  - [@patternfly/react-charts@7.2.1](https://www.npmjs.com/package/@patternfly/react-charts/v/7.2.1)
  - [@patternfly/react-code-editor@5.2.1](https://www.npmjs.com/package/@patternfly/react-code-editor/v/5.2.1)
  - [@patternfly/react-core@5.2.1](https://www.npmjs.com/package/@patternfly/react-core/v/5.2.1)
  - [@patternfly/react-drag-drop@5.2.1](https://www.npmjs.com/package/@patternfly/react-drag-drop/v/5.2.1)
  - [@patternfly/react-icons@5.2.1](https://www.npmjs.com/package/@patternfly/react-icons/v/5.2.1)
  - [@patternfly/react-styles@5.2.1](https://www.npmjs.com/package/@patternfly/react-styles/v/5.2.1)
  - [@patternfly/react-table@5.2.1](https://www.npmjs.com/package/@patternfly/react-table/v/5.2.1)
  - [@patternfly/react-tokens@5.2.1](https://www.npmjs.com/package/@patternfly/react-tokens/v/5.2.1)

### Fixes include:

> - Issues with styles and intractability of cards in Chromium based browsers

Reviewer: ```suggestion
- Issues with styles and inetractability of cards in Chromium based browsers
  - Additional details: https://github.com/orgs/patternfly/discussions/6381
```

- Support for status modifiers on Modal titles
- Type updates to react-charts to reflect changes in the Victory types
- Reversion of a previous type change with our clipboard copy component that was unintentionally breaking
- Pass through of more props to monaco-editor in our code editor component

> 

Reviewer: ```suggestion

<Divider />

```

import './get-started.css';
import { Divider } from '@patternfly/react-core';

## PatternFly 5.2.1

> PatternFly 5.2.1 is now live! This was a patch release to fix the below bugs. Here are the package versions from this release. 

Reviewer 2: ```suggestion
PatternFly 5.2.1 is now live! This was a patch release to fix a few bugs that were found.
```

### Promoted package versions

- patternfly/patternfly ([changelog](https://github.com/patternfly/patternfly/releases/tag/patch-5.2.1))
  - [@patternfly/patternfly@5.2.1](https://www.npmjs.com/package/@patternfly/patternfly/v/5.2.1)

- patternfly/patternfly-react ([changelog](https://github.com/patternfly/patternfly-react/releases/tag/5.2.1))
  - [@patternfly/react-charts@7.2.1](https://www.npmjs.com/package/@patternfly/react-charts/v/7.2.1)
  - [@patternfly/react-code-editor@5.2.1](https://www.npmjs.com/package/@patternfly/react-code-editor/v/5.2.1)
  - [@patternfly/react-core@5.2.1](https://www.npmjs.com/package/@patternfly/react-core/v/5.2.1)
  - [@patternfly/react-drag-drop@5.2.1](https://www.npmjs.com/package/@patternfly/react-drag-drop/v/5.2.1)
  - [@patternfly/react-icons@5.2.1](https://www.npmjs.com/package/@patternfly/react-icons/v/5.2.1)
  - [@patternfly/react-styles@5.2.1](https://www.npmjs.com/package/@patternfly/react-styles/v/5.2.1)
  - [@patternfly/react-table@5.2.1](https://www.npmjs.com/package/@patternfly/react-table/v/5.2.1)
  - [@patternfly/react-tokens@5.2.1](https://www.npmjs.com/package/@patternfly/react-tokens/v/5.2.1)

> ### Fixes include:
> - Issues with styles and inetractability of cards in Chromium based browsers
>   - Additional details: https://github.com/orgs/patternfly/discussions/6381
> - Support for status modifiers on Modal titles
> - Type updates to react-charts to reflect changes in the Victory types
> - Reversion of a previous type change with our clipboard copy component that was unintentionally breaking
> - Pass through of more props to monaco-editor in our code editor component

Reviewer 2: ```suggestion
### Fixed bugs
In this patch release, we:
- Corrected issues with styles and intractability of cards in Chromium-based browsers.
  - Additional details: https://github.com/orgs/patternfly/discussions/6381
- Added support for status modifiers on modal titles.
- Updated types used in react-charts to reflect changes in the Victory types.
- Reverted a previous type change within our clipboard copy component that was unintentionally breaking.
- Updated our code editor component to pass through more properties to monaco-editor. 
```
